db/seeds.rb

```diff
@@ -0,0 +1,12 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
+5.times do |i|
+    Greeting.create(
+      message: "Hello Tadesse #{i + 1}"
```

Comment:
- [ ] Good job working on this project, Kindly create 5 random greeting messages that will be displayed, currently I see just 1-5 when I refresh 👍🏿

Reply:
Thank you for the feedback I fix it per your request.